schema/query/projection_validator_test.go

```diff
@@ -50,6 +50,7 @@ func TestProjectionValidate(t *testing.T) {
 		{`with_params(foo:3.14)`, errors.New("with_params: invalid param `foo' value: not an integer")},
 		{`with_params(bar:1)`, errors.New("with_params: invalid param `bar' value: not a Boolean")},
 		{`with_params(foobar:true)`, errors.New("with_params: invalid param `foobar' value: not a string")},
+    {`*`, nil},
```

Comment:
Please add cases for:
* Fields with `*` in its name.
* `*` followed by valid fields.
* Valid fields followed by `*`.

Also, beware indentation. Are you using `gofmt`?

Reply:
I will add the cases. Indentation problem was due to resolving merge conflict with GitHub Web GUI.